_attributes/common-attributes.adoc

```diff
@@ -68,7 +68,7 @@ ifdef::openshift-origin[]
 endif::[]
 // Backup and restore
 :velero-domain: velero.io
-:velero-version: 1.9
+:velero-version: 1.12
```

Comment:
Here I believe is still 1.9 (from https://github.com/openshift/oadp-operator#velero-version-relationship), but I would also like the input from someone with more time in the project